include/zephyr/drivers/sensor.h

```diff
@@ -418,7 +418,7 @@ typedef int (*sensor_channel_get_t)(const struct device *dev,
  * @brief Decodes a single raw data buffer
  *
  * Data buffers are provided on the @ref rtio context that's supplied to
- * c:func:`sensor_read`.
+ * :c:func:`sensor_read`.
```

Comment:
This won't work in a doxygen comment and should be a doxygen `@ref` instead

Reply:
That is true will fix it I was mislead by https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/drivers/sensor_data_types.h